website/content/docs/nia/cli/index.mdx

```diff
@@ -11,71 +11,71 @@ Consul-Terraform-Sync (CTS) is controlled via an easy to use command-line interf
 
-Flag: `-inspect`
+To get help for a command, run: `consul-terraform-sync <command> -h`
 
-Behavior: CTS will display the proposed state changes for all tasks once and exit. No changes are applied in this mode. On encountering an error before completing, CTS will exit with a non-zero status.
+### CLI Structure
 
-Usage: Intended to be run before daemon-mode in order to confirm configuration is accurate and tasks would update network infrastructure as expected.
+CTS commands follow the below structure
 
----
+```shell-session
+consul-terraform-sync <command> [options] [args]
+```
 
-Flag: `-inspect-task [task-name]`
+- `options`: Flags to specify additional settings. There are general options that can be used across all commands and command-specific options.
+- `args`: Required arguments specific to a commands
 
-Behavior: This has similar behavior as `-inspect` mode for the selected task. The flag can be specified multiple times to inspect multiple tasks. No changes are applied in this mode.
+Example:
 
-Usage: Useful to debug one or more tasks to confirm configuration is accurate and the selected tasks would update network infrastructure as expected.
+```shell-session
+consul-terraform-sync task disable -http-addr=http://localhost:2000 task_a
+```
 
-#### Once Mode
+### Autocompletion
 
-Flag: `-once`
+The `consul-terraform-sync` command features opt-in autocompletion for flags, subcommands, and
+arguments (where supported).
 
-Behavior: CTS will run all tasks once with buffer periods disabled and exit. On encountering an error before completing, CTS will exit with a non-zero status.
+To enable autocompletion, run:
+```shell-session
+$ consul-terraform-sync -autocomplete-install
+```
 
-Usage: Intended to be run before daemon-mode in order to confirm configuration is accurate and tasks update network infrastructure as expected.
+~> After you install autocomplete, you must restart your shell for the change to take effect.
 
-## Commands
+When you start typing a CTS command, press the `<tab>` key to show a
+list of available completions. To show available flag completes, type `-<tab>`.
 
-In addition to running the daemon, CTS has a set of commands that act as a client to the daemon server. The commands provide a user-friendly experience interacting with the daemon. The commands use the CTS APIs but does not correspond one-to-one with it. Please see the individual commands in the left navigation for more details.
+Autocompletion will query the running CTS server to return helpful argument suggestions. For example, for the `task disable` command, autocompletion will return the names of all enabled tasks that can be disabled.
 
-To get help for a command, run: `consul-terraform-sync <command> -h`
+When autocompletion makes the query to the running CTS server, it will also use any `CTS_*`` environment variables (for example `CTS_ADDRESS`) set on the CTS server.
```

Comment:
```suggestion
When autocompletion makes the query to the running CTS server, it will also use any `CTS_*` environment variables (for example `CTS_ADDRESS`) set on the CTS server.
```

Reply:
Ah, I had also added a fix for this in the pr to be merged in afterwards https://github.com/hashicorp/consul/pull/13191. This works!